app/scripts/lib/auth-errors.js

```diff
@@ -56,7 +56,7 @@ define(function (require, exports, module) {
     },
     INCORRECT_PASSWORD: {
       errno: 103,
-      message: t('Incorrect password')
+      message: t("Incorrect password.To double-check what you've\nentered, press and hold down the Show button.")
```

Comment:
1. need space after the `.` in `password. To double`
2. `you've\nentered` what happened there?
3. use `'` instead of `"` (single quotes)